friends/src/App.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+  
+    this.state = {
+    friends: [],
+    name: '',
+    age: '',
+    email: '',
+  };
+}
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends')
+      .then(response => {
+        this.setState({ friends: response.data});
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
+  handleChange = event => {
+    this.setState({[event.target.name]: event.target.value})
+  }
+
+  saveFormData = () => {
+    const id = (this.state.friends[this.state.friends.length -1].id +1)
```

Comment:
the back-end handles the `id`s for you, so you don't have to worry about it 🙅‍♂️ 😟